tests/helm/testdata/goldenfile/traces-gateway-deployment/custom.output.yaml

```diff
@@ -41,16 +48,27 @@ spec:
                 secretKeyRef:
                   name: sumologic
                   key: endpoint-traces
+            - name: SUMO_ENDPOINT_DEFAULT_OTLP_TRACES_SOURCE
```

Comment:
this is here because we enabled the otlp source in the input, correct?

Reply:
Yes.